_posts/2019-07-24-cockpit-199.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cockpit 199
+author: kkoukiou
+date: 2019-07-24
+tags: cockpit linux
+slug: cockpit-199
+category: release
+summary: Cockpit with Logs and Services and HWInfo design updates
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 199.
+
+### New logs design
+
+Restyled logs across all of Cockpit.
+
+![service-logs](/images/service-logs.png)
+
+### Design and accesibility improvements on Service page
```

Comment:
Let's please continue the tradition of "Page: Description", i. e "Services: Design and accesibility improvements". This is a bit different for the logs design as it affects more than one page.